react/components/atoms/lightbox/lightbox.js

```diff
@@ -106,37 +342,59 @@ export class Lightbox extends PureComponent {
                     visible={this.state.visible}
                     onRequestClose={this.onBackButtonPress}
                 >
-                    <View style={styles.fullscreenContainer}>
-                        <Image style={this._imageFullscreenStyle()} source={this._imageSource()} />
-                        <ButtonIcon
-                            icon={"close"}
-                            onPress={this.onClosePress}
-                            style={styles.buttonClose}
-                            iconStrokeWidth={2}
-                            size={isTabletSize() ? 52 : 34}
-                            iconHeight={isTabletSize() ? 34 : 22}
-                            iconWidth={isTabletSize() ? 34 : 22}
-                            backgroundColor={"#000000"}
-                            iconStrokeColor={"#ffffff"}
-                        />
-                    </View>
+                    <Animated.View style={styles.fullscreenContainer}>
+                        <PanGestureHandler

```

Comment:
It seems the removal of `GestureHandlerRootView` causes this not to work on Android, even by adding `GestureHandlerRootView` to the app.js of robin, for example. For it to work it would require the usage of `gestureHandlerRootHOC`, like it says in the [documentation](https://docs.swmansion.com/react-native-gesture-handler/docs/#usage-with-modals-on-android). I tried and there is some flickering happening when zooming so perhaps maybe reverting this change (I'm sorry, I know I was the one to tell you to remove it). What do you think?